source/StigData/Processed/Office-365ProPlus-2.3.xml

```diff
@@ -1342,7 +1344,7 @@ Use the Windows Registry Editor to navigate to the following key:
 
 HKCU\software\policies\microsoft\office\16.0\excel\security
 
-If value for extensionhardening is REG_DWORD = 1, this is not a finding.</RawString>
+If value for extensionhardening is REG_DWORD = 2, this is not a finding.</RawString>
```

Comment:
This is a hard coded rule that is no longer correct based on guidance.

Reply:
For V-223332 the hard coded rule is:

V-223332::*::HardCodedRule(RegistryRule)@{DscResource = 'RegistryPolicyFile'; Key = 'HKEY_CURRENT_USER\Software\Policies\Microsoft\Office\16.0\Excel\Security'; ValueData = "1"; ValueName = 'extensionhardening'; ValueType = 'Dword'}

Reply:
according to the check content the value should be 2 not 1, or did I miss something?